src/lib/env/_user-env.js

```diff
@@ -83,6 +83,17 @@ module.exports = function populateUserEnv (params, callback) {
       return callback(error)
     }
   }
+  else if (env) {
+    Object.entries(env).forEach(entry => {
+      let [ key, value ] = entry
+      if (typeof value === 'string') userEnv[key] = value
+      else if (typeof value === 'undefined') delete userEnv[key]
+      else {
+        let error = `env option '${key}' parse error: ${new Error().stack}`
+        return callback(error)
+      }
+    })
```

Comment:
Probably needs to be added here for logging purposes:
`print('testing', 'env option')`